fixtures/cameo/nanospot-120.json

```diff
@@ -139,53 +135,90 @@
     },
     "Program": {
       "type": "Effect",
+      "defaultValue": 0,
```

Comment:
In most plugins, this will result in fallbacking to the default channel, which is `No function`. I'd suggest replacing the `No function` channel usages with `Program Speed`, so that a usable channel will be exported.